### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by blackcoffee2525** [HMBSbige/BilibiliLiveRecordDownLoader#297] [General]
你好，问一个问题，如果在下载直播推流时，自己的网络因为各种原因断了，导致断流前录上的视频未能完全下载完全，而且无法打开，有什么解决办法吗
![9$~38V9{Y} 1%YB1)W6T F9](https://github.com/HMBSbige/BilibiliLiveRecordDownLoader/assets/140333477/4e4834a7-b494-4ee6-a1ee-c72312963860)

**Comment by HMBSbige** [comment]
截图也不截全...我都不知道你什么格式
不过所有格式都是分段下载的，所以就算各种原因中断也还是可以打开的
如果遇到时间戳混乱可以尝试转成mp4

**Reply by blackcoffee2525** [reply]
我把截图补一下，简单来说就是，第二张截图里那个7月20号的因为网络原因断了，导致当时下的视频打开不了，你说各种原因中断也可以打开，能不能帮我看看是不是我的配置有问题
![~PO$E%{J1RO2B HM{4OV`TY](https://github.com/HMBSbige/BilibiliLiveRecordDownLoader/assets/140333477/a3c4b204-d153-4327-ada3-8b6d17101dbd)
![H 33E$ZKND`~K`R$Z( }2VA](https://github.com/HMBSbige/BilibiliLiveRecordDownLoader/assets/140333477/a31ce434-9cee-491f-a4b7-88e837d807a4)
![LY@H`% 91 PHK}V {@DM)2B](https://github.com/HMBSbige/BilibiliLiveRecordDownLoader/assets/140333477/79bdb8ce-c8c4-4aba-adf0-0fa480c287e9)

**Reply by HMBSbige** [reply]
看样子你已经转了mp4，如果你还存留原始下载的视频的话可能还能帮到你

**Comment by blackcoffee2525** [comment]
![H 33E$ZKND`~K`R$Z( }2VA](https://github.com/HMBSbige/BilibiliLiveRecordDownLoader/assets/140333477/cac39aa7-93b9-49b1-962b-1457a6c0af7b)
![~PO$E%{J1RO2B HM{4OV`TY](https://github.com/HMBSbige/BilibiliLiveRecordDownLoader/assets/140333477/0f3d18de-a4e3-44ed-9bae-c26b987b1726)
